### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by wanghaoran1988**
**What this PR does / why we need it**:
Fix typo, should be single "="

**Release note**:
```
None
```

### CODE REVIEW

**Comment by wanghaoran1988** [comment]
@bsalamat

File: plugin/cmd/kube-scheduler/app/options/options.go [hunk 1]
```diff
@@ -63,8 +63,8 @@ func (s *SchedulerServer) AddFlags(fs *pflag.FlagSet) {
 	fs.Int32Var(&s.Port, "port", s.Port, "The port that the scheduler's http service runs on")
 	fs.StringVar(&s.Address, "address", s.Address, "The IP address to serve on (set to 0.0.0.0 for all interfaces)")
 	fs.StringVar(&s.AlgorithmProvider, "algorithm-provider", s.AlgorithmProvider, "The scheduling algorithm provider to use, one of: "+factory.ListAlgorithmProviders())
-	fs.StringVar(&s.PolicyConfigFile, "policy-config-file", s.PolicyConfigFile, "File with scheduler policy configuration. This file is used if policy ConfigMap is not provided or --use-legacy-policy-config==true")
```
Comment (bsalamat) [inline plugin/cmd/kube-scheduler/app/options/options.go]: I am fine with the change, but these were intentional. I meant to say "if" --use-legacy-policy-config is true.

**Comment by cblecker** [comment]
@k8s-bot ok to test

File: plugin/cmd/kube-scheduler/app/options/options.go (hunk 1 above)
Comment (wanghaoran1988) [inline plugin/cmd/kube-scheduler/app/options/options.go]: @bsalamat It's ok if were intentinal, just make me/user confused, as  when we run the command, the option should be used with single "=".

Comment (bsalamat) [inline plugin/cmd/kube-scheduler/app/options/options.go]: Sure, I have no strong preference.

**Pull request closed without merging**